keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -52,7 +53,7 @@ class ArtifactController(
   @ResponseStatus(CREATED)
   fun register(@RequestBody artifact: DeliveryArtifact) {
     log.debug("Registering {} artifact {}", artifact.type, artifact.name)
-    artifactRepository.register(artifact)
```

Comment:
I've moved this into the `ArtifactListener` because I thought it made sense to centralize registering an artifact.